block/manager.go

```diff
@@ -566,17 +560,13 @@ func (m *Manager) getCommit(header types.Header) (*types.Commit, error) {
 func (m *Manager) IsProposer() (bool, error) {
 	m.lastStateMtx.RLock()
```

Comment:
i don't think you need to lock here as you no longer use lastState?

Reply:
Fixed